crowbar_framework/config/application.rb

```diff
@@ -19,7 +19,7 @@
 
 module Crowbar
   class Application < Rails::Application
-    SERVER_PID = system("[ps ax | grep 'puma master' | grep -v grep].split(' ')[0]")
+    SERVER_PID = %x[pgrep puma | sort | head -n1].strip
```

Comment:
Why the `sort`? I don't think there's a guarantee that the main process is the first one...